get-involved.php

```diff
@@ -40,49 +40,40 @@
 <section id="funding" class="grey">
     <div class="grid">
         <div class="two-thirds">
-            <h1>Funding</h1>
+            <h2>Funding</h2>
```

Comment:
I agree with this change, but now it doesn't match the other sections. Should we update the other sections to have h2 headings and h3 subheadings?